src/ensembl/src/shared/components/help-article/HelpArticle.scss

```diff
@@ -0,0 +1,41 @@
+@import 'src/styles/common';
+@import 'helpArticleConstants';
+
+.textArticle {
+
+  h1 {
+    font-size: 15px;
+    line-height: 18px;
+    font-weight: $normal;
+    margin: 0 0 #{$heading-margin-bottom} -#{$heading-outdent};
+  }
+
+  h2 {
+    font-size: 13px;
+    margin-top: calc(2 * #{$line-height});
+    margin-bottom: $line-height;
+  }
+
+  p {
+    line-height: $line-height;
+    margin: $line-height 0;
+  }
+
+  img {
+    max-width: 100%;
+  }
+
+  // Notice that the external-link svg has the same shape as in the static/img/shared folder
+  // But the version copied here has the fill color set inline, because it is being accessed as an external image via url
+  a[href^=http] {
```

Comment:
Won't these catch internal links as well?

Reply:
Depends. If an internal link is just a link to another help article, it will likely be written in the markdown source file like this: `[Another article]('another-article.md')`, which will get transformed into something like `/help/articles/another-article`, and thus won't have a protocol in it.

Links to other sections of the site can also be written without protocol and host (e.g. `/browser`, `/entity-viewer`). This will leave the full urls, with protocol and host written out, only for truly external links.